frontend/src/Components/ProgressBar.css

```diff
@@ -23,6 +23,10 @@
   color: var(--white);
 }
 
+.backTextContainer {
+  color: var(--white);
```

Comment:
It's different colours so it shows up on the background for light theme properly as well as the various bar colours. This change completely breaks the light theme.

![image](https://user-images.githubusercontent.com/413544/206951046-b38eac9d-b8bb-426f-a7e3-3de906ab9e48.png)

Reply:
instead of setting this to a color like `--white` maybe helpful to move to a variable like `--progressBarBackgroundColor` or set of variables for progress bar text color. This will allow it to be different per theme.